README.rst

```diff
@@ -60,7 +61,7 @@ Usage
       # Run axe accessibility checks.
       results = axe.execute()
       # Write results to file
-      axe.write_results('a11y.json', results)
+      axe.write_results(results, 'a11y.json')
```

Comment:
👍

Reply:
@mudgil glad to hear it! Thank you for your contribution 🙂